sos/plugins/openstack_trove.py

```diff
@@ -34,21 +35,13 @@ def setup(self):
     def postproc(self):
 
         protect_keys = [
-            "dns_passkey", "nova_proxy_admin_pass", "rabbit_password",
-            "qpid_password", "connection", "sql_connection", "admin_password"
-        ]
-
-        conf_list = [
-            '/etc/trove/trove.conf',
-            '/etc/trove/trove-conductor.conf',
-            '/etc/trove/trove-guestmanager.conf',
-            '/etc/trove/trove-taskmanager.conf'
+            "default_password_length", "notifier_queue_password",
+            "rabbit_password", "replication_password", "connection",
+            "admin_password", "dns_passkey"
```

Comment:
You are removing from protect_keys these params/items:

nova_proxy_admin_pass
qpid_password
sql_connection

Is the removal safe or can the parameters have values/passwords to be obfuscated?